wagtail_content_import/pickers/google/utils.py

```diff
@@ -14,6 +14,9 @@ def parse_document(document):
 
 
 class GooglePicker(Picker):
+    source = "google"
```

Comment:
Shall we call these `name`/`verbose_name` for consistency with Django [app configs](https://docs.djangoproject.com/en/2.2/ref/applications/#for-application-authors)?

Reply:
That makes sense, I've changed now.